README.md

```diff
@@ -22,11 +22,13 @@ This module is really just about making it work with Nuxt 3 without the need to
 
 ## Quick Setup
 
+Run `npx` command to add `nuxt3-leaflet` into your `package.json` and `nuxt.config.ts`
```

Comment:
@danielroe relatives to https://github.com/fuentesloic/nuxt-stripe/pull/28#discussion_r1560384387